docs/content/Modpacks/Materials-and-Elements/Material-Flags.md

```diff
@@ -8,60 +8,218 @@ title: Material Flags
+
+- `DECOMPOSITION_BY_CENTRIFUGING`
+    - Description: Enables centrifuge decomposition recipe generation (Requires componets to be set).
+
+- `DISABLE_DECOMPOSITION`
+    - Description: Disables decomposition recipe generation for this material.
+
+- `EXPLOSIVE`
+    - Description: Any material with this flag wont have implosion compression recipes, and it will give ash when you arc furnace recycle it instead of that material.
+
+- `FLAMMABLE`
+    - Description: Adding this flag means you cant smelt that material (it wont generate a ebf recipe/furnace recipe) also disables impossion compressor recipes like explosive does.
+
+- `STICKY`
+    - Description: Add to material if it is sticky (Changes the viscosity of the placed fluid. Only the oils and creosote have a placeable state by default).
+
+- `PHOSPHORESCENT`
+    - Description: Adding this flag onto a material gives liquids a luminosity of 15, no matter the fluid state(liquid, gas, plasma) otherwise they default to 10 for specifically liquid state.
+
+# Dust Flags
+
+- `GENERATE_PLATE` 
+     - Description: Generates a plate and double plate for this material.
+     - Required Flags: `GENERATE_PLATE`.
+     - Required Properties: `PropertyKey.DUST`.
+
+- `GENERATE_DENSE` 
+     - Description: Generates a dense plate for this material.
+     - Required Flags: `GENERATE_PLATE`.
+     - Required Properties: `PropertyKey.DUST`.
+
+- `GENERATE_ROD`
+    - Description: Generates a rod for this material.
+    - Required Properties: `PropertyKey.DUST`.
 
-- `FORCE_GENERATE_BLOCK`
 - `GENERATE_BOLT_SCREW`
-- `GENERATE_DENSE`
-- `GENERATE_FINE_WIRE`
-- `GENERATE_FOIL`
+    - Description: Generates a bolt and screw for this material.
+    - Required Flags: `GENERATE_ROD`.
+    - Required Properties: `PropertyKey.DUST`.
+
 - `GENERATE_FRAME`
+    - Description: Generates a frame for this material.
+    - Required Flags: `GENERATE_ROD`.
+    - Required Properties: `PropertyKey.DUST`.
+
 - `GENERATE_GEAR`
-- `GENERATE_LENS`
+    - Description: Generates a gear for this material.
+    - Required Flags: `GENERATE_PLATE`, `GENERATE_ROD`.
+    - Required Properties: `PropertyKey.DUST`.
+
 - `GENERATE_LONG_ROD`
-- `GENERATE_PLATE`
-- `GENERATE_RING`
-- `GENERATE_ROD`
-- `GENERATE_ROTOR`
-- `GENERATE_ROUND`
-- `GENERATE_SMALL_GEAR`
-- `GENERATE_SPRING`
-- `GENERATE_SPRING_SMALL`
+    - Description: Generates a long rod for this material.
+    - Required Flags: `GENERATE_ROD`.
+    - Required Properties: `PropertyKey.DUST`.
+
+- `FORCE_GENERATE_BLOCK`
+    - Description: Add this to a material to force generate a block.
+    - Required Properties: `PropertyKey.DUST`.
 
+- `EXCLUDE_BLOCK_CRAFTING_RECIPES`
+    - Description: This will prevent material from creating Shapeless recipes for dust to block and vice versa. Also preventing extruding and alloy smelting recipes via SHAPE_EXTRUDING/MOLD_BLOCK.
+    - Required Properties: `PropertyKey.DUST`.
```

Comment:
```suggestion
    - Description: This will prevent material from creating Shapeless recipes for dust to block and vice versa. Also preventing extruding and alloy smelting recipes via `SHAPE_EXTRUDING`/`MOLD_BLOCK`.
    - Required Properties: `PropertyKey.DUST`.
```